test/index.spec.js

```diff
@@ -467,34 +467,32 @@ describe('daemons', () => {
     })
 
     it('starts the daemon and returns valid API and gateway addresses', (done) => {
-      let daemon
       const dir = `${os.tmpdir()}/tmp-${Date.now() + '-' + Math.random().toString(36)}`
 
       async.waterfall([
         (cb) => ipfsd.local(dir, cb),
-        (node, cb) => {
-          daemon = node
-          node.init((err) => cb(err, node))
```

Comment:
Seems like in Node.js 8 a copy was being made instead of a reference.. WAT?

Reply:
Hmm that sounds dubious, we should change the test, if this is making copies there is likely a larger issue at play which will be a problem for users of ipfsd-ctl

Reply:
It seems that this didn't fixed it after all, I just got lucky

Reply:
got it